Looms.PoS/Looms.PoS.Application/Utilities/Validators/FluentValidationExtensions.cs

```diff
@@ -10,4 +10,23 @@ public static IRuleBuilder<T, string> MustBeValidGuid<T>(this IRuleBuilder<T, st
                       .Must(id => Guid.TryParse(id, out _) && id.Length == 36)
                       .WithMessage("{PropertyName} is not a valid Guid");
     }
+
+    public static IRuleBuilder<T, string> MustBeValidDateTime<T>(this IRuleBuilder<T, string> builder)
+    {
+        return builder.NotEmpty()
+                      .Must(date => DateTime.TryParse(date, out _)) // should we change to specific format?
```

Comment:
If i remember correctly Aiste wrote a specific format somewhere in code

Reply:
"yyyy-MM-dd HH:mm:ss" in Application/Utilities/DateTimeHelper